src/test/java/com/qaprosoft/carina/demo/SoapSampleTest.java

```diff
@@ -0,0 +1,36 @@
+package com.qaprosoft.carina.demo;
+
+import com.qaprosoft.apitools.validation.XmlCompareMode;
+import com.qaprosoft.carina.core.foundation.IAbstractTest;
+import com.qaprosoft.carina.demo.soap.AddIntegerMethod;
+import com.qaprosoft.carina.demo.soap.LookupCityMethod;
+import io.restassured.path.xml.XmlPath;
+import io.restassured.response.Response;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class SoapSampleTest implements IAbstractTest {
+
+    @Test
+    public void testAddInteger() {
+        AddIntegerMethod soap = new AddIntegerMethod();
+        soap.setProperties("api/soap/soap.properties");
+        soap.setHeaders(String.format("SOAPAction=%s", "http://tempuri.org/SOAP.Demo.AddInteger"));
```

Comment:
that's better to move setHeaders() call into AddIntegerMethod as it's default action for this method